osu.Game.Rulesets.Osu/Mods/OsuModTarget.cs

```diff
@@ -51,14 +51,26 @@ public class OsuModTarget : ModWithVisibilityAdjustment, IApplicableToDrawableRu
 
         public bool DisplayResultsOnFail => true;
 
-        // Maximum distance to jump
-        private const float max_distance = 250f;
+        /// <summary>
+        ///     Jump distance for circles in the last combo
+        /// </summary>
+        private const float max_base_distance = 250f;
+
+        /// <summary>
+        ///     The maximum allowed jump distance after multipliers are applied
```

Comment:
```suggestion
        /// The maximum allowed jump distance after multipliers are applied
```

Incorrect spacing, also applies to all other xmldoc.

Reply:
Already fixed. I had the wrong coding style setting in Rider.